src/pages/docs/projects/project-triggers/git-triggers.md

```diff
@@ -0,0 +1,74 @@
+
+Git repositories referenced in your project's deployment process can be selected to be monitored by the trigger to create releases.
+
+Please note that for [configuration as code](/docs/projects/version-control/config-as-code-reference) projects, only steps that reference Git repositories in the deployment process from the **default branch** are able to be referenced. Any changes to the deployment process in other branches will not be available for use in git triggers.
+
+:::figure
+![Repository selection](/docs/projects/project-triggers/images/git-triggers/git-triggers-repository-selection.png)
+:::
+
+### File path filters
+
+When selecting a repository to monitor you will be provided with the option to add file path filters. These filters allow you to specify file paths to include or exclude from the monitoring of new commits.
+
+:::figure
+![File path filters](/docs/projects/project-triggers/images/git-triggers/git-triggers-file-path-filters.png)
+:::
+
+- If no file path filters are specified, all commits to the monitored repository will trigger the creation of a new release. 
+- If file paths are set to be included, only changes to those file paths will be monitored, all other file paths will be excluded.
+- If file paths are set to be excluded, changes to those file paths will not be monitored, all other file paths will be included.
+
+The file path filters support glob patterns and can include the following wildcard characters:
+
+| **Character** | **Description**                           | **Example**                                                                                                           |
+|---------------|-------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| `*`           | Matches multiple characters except `/`    | File path pattern of `release/*` will match the file path `release/1.0.0` but not `release/1.0.0/hotfix1`             |
+| `**`          | Matches multiple characters including `/` | File path pattern of `release/**` will match the file paths `release/1.0.0` and `release/1.0.0/hotfix1`               |
+| `?`           | Matches a single character                | File path pattern of `v?` will match a file path of `v1` but not `v1.0.0`                                             |
+| `[0-9]`       | Matches a single character in the range   | File path pattern of `v[0-9].[0-9].[0-9]` will match the file path `v1.0.0`                                           |
+| `[abc]`       | Matches a single character from the set   | File path pattern of `release/[abc]*` will match the file path `release/a-new-branch` but not `release/my-new-branch` |
+
+## History
+
+The history section contains information about the last time the trigger was evaluated and the last release that was created by the trigger. By default, triggers are evaluated every three minutes and the results will be reported here.
```

Comment:
```suggestion
The history section contains information about the last time the trigger was evaluated and the last release that was created by the trigger. Triggers are evaluated every three minutes and the results will be reported here.
```
Having "By default" here makes it sound like its configurable which from memory it's not?

Reply:
Removed